src/postgres/tree/PostgresServerTreeItem.ts

```diff
@@ -68,15 +75,30 @@ export class PostgresServerTreeItem extends AzureParentTreeItem<ISubscriptionCon
     }
 
     public async loadMoreChildrenImpl(_clearCache: boolean): Promise<AzExtTreeItem[]> {
-        const client: PostgreSQLManagementClient = createAzureClient(this.root, PostgreSQLManagementClient);
-        const listOfDatabases: DatabaseListResult = await client.databases.listByServer(this.resourceGroup, this.name);
+        if (!(this.connectionString.username && this.connectionString.password)) {
+            const { username, password } = await this.getCredentials();
+            if (username && password) {
+                await this.updateConnectionString(username, password);
+            }
+        }
+        let config;
+        let query: string;
+        if (this.connectionString.databaseName) {
+            config = await getClientConfig(this, this.connectionString.databaseName);
+            query = `SELECT datname FROM pg_catalog.pg_database WHERE datistemplate = false and datname = '` + nonNullProp(this.connectionString, 'databaseName') + `';`;
```

Comment:
I'd like you to make a function like "wrapArgInQuotes" (maybe put it in "runPostgresQuery.ts"?) so that our quoting logic is centralized. It'll ensure we're consistent, but also make it easier to change the behavior for all queries at once. Quoted identifiers are case sensitive and unquoted identifiers are case insensitive - that doesn't feel like something where one option will satisfy all users. In other words, "wrapArgInQuotes" may be a one-liner in this PR, but I don't expect it to stay that way